server/datastore/mysql/statistics.go

```diff
@@ -84,6 +84,12 @@ func (ds *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Du
 		stats.VulnDetectionEnabled = appConfig.VulnerabilitySettings.DatabasesPath != ""
 		stats.SystemUsersEnabled = appConfig.Features.EnableHostUsers
 		stats.HostsStatusWebHookEnabled = appConfig.WebhookSettings.HostStatusWebhook.Enable
+		stats.MDMEnabledAndConfigured = appConfig.MDM.EnabledAndConfigured
+		stats.HostExpiryEnabled = appConfig.HostExpirySettings.HostExpiryEnabled
+		// stats.JiraIntegrationEnabled = //How do we get those? Remove if con't find
```

Comment:
What do you mean by "How do we get those?"?

There's `appConfig.Integrations.Jira` and `appConfig.Integrations.Zendesk` (If one of those slices has more than one item then it means such integration is configured.):

```go
stats.JiraIntegrationEnabled = len(appConfig.Integrations.Jira) > 0
stats.ZendeskIntegrationEnabled = len(appConfig.Integrations.Zendesk) > 0
```

Alternative instead of a bool we can store an int to represent how many integrations of each are configured?

```go
stats.JiraIntegrationsCount = len(appConfig.Integrations.Jira)
stats.ZendeskIntegrationsCount = len(appConfig.Integrations.Zendesk)
```

Reply:
That's the answer I needed 🙂 
Thanks.

Reply:
BTW - I prefer the bools because we will accumulate the numbers and not present it per org.
With bools we'll be able to tell how many orgs use it.

@eashaw If you have an idea how to count the integrations we can change it later.